holidayextras/holidayextras.github.com
# Changing Gravatar plugin to md5

## What is the problem?
- Plugins aren't accepted on GitHub so our Gravatar plugin doesn't work live (but works locally, hence no issue found in testing)

## What does this PR do?
- Removes the plugin for Gravatar and instead calls each team members MD5 value so the image is shown live

## How to test
- Checkout branch
- Run `jekyll serve` this should boot up the blog locally (if not try `bundle exec jekyll s`)
- Look at the team page and decided if it looks ok to release LIVE to the world
- Check a few blog posts for photos
- ENSURE Photos are shown or the little HX green man.

## Review
### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
